slack_bot/main.py

```diff
@@ -1,6 +1,13 @@
-"""Application entry point"""
-from fastapi import FastAPI
-from .slash_commands import slash_commands_router
+"""Asyncronious events listener"""
+from fastapi import FastAPI, Request
+from slack_bot.bot import app_handler
+from slack_bot.run_ngrok import setup_ngrok
 
-app = FastAPI()
-app.include_router(slash_commands_router)
+setup_ngrok()
+fast_app = FastAPI()
+
+
+@fast_app.post("/slack/events")
+async def handle_events(req: Request):
+    """Asyncronious events listener of slack bot events"""
+    return await app_handler.handle(req)
```

Comment:
Are there any events expected to be handled? 
Asking since this may conflict with my changes in #20

Reply:
kind of, earlier we had `/process_command` endpoint that handled all commands, now I've just changed commands request url to events endpoint and let Bolt handle it.